linear-baking-kernighan.md

```
@@ -0,0 +1,131 @@
+| Monitor update utility | `src/main/kotlin/com/codescene/jetbrains/platform/webview/util/UpdateMonitor.kt` |
+| Saved files tracker | `src/main/kotlin/com/codescene/jetbrains/platform/git/SavedFilesTrackerAdapter.kt` |
+| Open files observer | `src/main/kotlin/com/codescene/jetbrains/platform/git/OpenFilesObserverAdapter.kt` |
+| Git change lister | `src/main/kotlin/com/codescene/jetbrains/platform/git/Git4IdeaChangeLister.kt` |
+| Canonical VS Code impl | `~/ext/src/git/git-change-observer.ts` |
+| Canonical VS Code tests | `~/ext/src/test/suite/git-change-observer.test.ts` |
+
+---
+
+## Data Flow
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)